Resources/doc/reference/installation.rst

```diff
@@ -192,6 +192,10 @@ Add the related security routing information:
         resource: '@SonataUserBundle/Resources/config/routing/admin_security.xml'
         prefix: /admin
 
+    sonata_user_admin_resetting:
```

Comment:
The both are not the same. `sonata_user_resetting` is for the frontend, `sonata_user_admin_resetting` is for the admin reset.